modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/dht/topology/PartitionEvictionOrderTest.java

```diff
@@ -131,6 +135,8 @@ public void testSyncCachesEvictedAtFirst() throws Exception {
 
         ReentrantLock lock = new ReentrantLock();
 
+        CopyOnWriteArrayList<Integer> evictionOrder = new CopyOnWriteArrayList<>();
```

Comment:
Why CopyOnWriteArrayList ?
Complexity of its fill is O(N^2), where N is number of partitions. Collections.synchronizedList(new ArrayList<>()) would just O(N).